tests/pages/vendor-search.js

```diff
@@ -93,5 +93,14 @@ export default {
 
   clickBackButton() {
     return $('[data-test-eholdings-package-details-back-button]').trigger('click');
+  },
+
+  scrollToOffset(readOffset) {
+    let $list = $('[data-test-query-list="vendors"]').get(0);
+    let rowHeight = $('li', $list).get(0).offsetHeight;
+    let scrollOffset = rowHeight * readOffset;
+
+    $list.scrollTop = scrollOffset;
+    $list.dispatchEvent(new Event('scroll'));
```

Comment:
not worth pulling out just for this, but maybe down the line we could have a shared helpers file for page objects

Reply:
Hopefully that's where bigtest can come help :D